### ISSUE TITLE
{Linked:Bug9474657} [Visual Requirements - MyHub Viva UX]:  In 400% zoom mode, content and functional elements present in the page are not completely visible and not accessible.

### ISSUE THREAD

**Issue by Raisul123** [microsoft/AdaptiveCards#8109]
### Target Platforms

Other

### SDK Version

https://adaptivecards.io/ 

### Application Name

Adaptive Cards

### Problem Description

PWD Impact:
Users with low vision who rely on the large text will be impacted if the content and functional elements present in the page are not completely visible and not accessible.

Test Environment:
OS: Microsoft Windows 11 Enterprise (Version:21H2 Build 22000.318)
Browser: Microsoft Edge:Version 96.0.1054.29 (Official build) (64-bit)
Google Chrome: Version 98.0.4758.102 (Official Build) (64-bit)
URL:[https://microsoft.sharepointdf.com/teams/dev_myhubring0/SitePages/Dashboard.aspx](https://microsoft.sharepoint-df.com/teams/dev_myhubring0/SitePages/Dashboard.aspx)

Repro Steps:
Step 1: Open the URL in the Edge Chromium browser.Home page will be displayed.
Step 2:Navigate through the page using tab key.
Step 3: Click on 'Clear your calendar' button.
Step 4: Change screen resolution to 1280 X 1024.Increase browser zoom to 400%.
Step 5: Navigate through 'Select dates' dialog box using tab key and open datepicker.
Step 6: Verify whether the content and functional elements in datepicker are completely visible and accessible or not.

Instances:
1.Add receipient page and done page-> The content is overlapped and not clearly visible.
2.Homepage->when focus is on 'Clear your calendar'->verify content is properly visible or not.
3.Home->Clear your calendar->Next->Next->Manage calendar->Verify content is completely visible or not.

Note:
Same issue observed in chrome browser also.

Actual Result:
 In 400% zoom mode, content and functional elements present in the page are not completely visible and not accessible.

Expected Result:
 In 400% zoom mode, content and functional elements present in the page should completely visible and accessible with keyboard.

WCAG Reference:
https://www.w3.org/WAI/WCAG21/Understanding/reflow

### Screenshots

![Content not visible at 400 resize](https://user-images.githubusercontent.com/23114460/206196420-1f9e144f-01f4-480e-9422-2930956f9b0f.jpg)
<img width="622" alt="8143-MAS 1 4 10- In 400% zoom mode, content and functional elements present in the page are not completely visible and not accessible  (2)" src="https://user-images.githubusercontent.com/23114460/206196834-d81931df-cc93-4111-9306-be99f2ad85e2.png">
![MAS 1 4 10 Content is cropping](https://user-images.githubusercontent.com/23114460/206196872-05e9d618-25c8-4357-8161-c58a61b1a7fb.png)
![MAS 1 4 10 Content is visible clearly](https://user-images.githubusercontent.com/23114460/206196903-e387aeb6-a79c-4416-b0c7-14121e81cdc6.png)
<img width="921" alt="MAS 1 4 10 In 400% zoom, the content is overlapped and not clearly visible" src="https://user-images.githubusercontent.com/23114460/206196907-b1a0cf88-93d7-434e-a6c6-cdbeafec7367.png">
![MAS 1 4 10 When the focus is on clear your calendar, content is cropping](https://user-images.githubusercontent.com/23114460/206196916-557db3cd-f54e-460e-9eff-65b815322232.png)

### Card JSON

```JSON
{
    "schema": "http=//adaptivecards.io/schemas/adaptive-card.json",
    "type": "AdaptiveCard",
    "version": "1.2",
    "body": [
        {
            "id": "",
            "isVisible": true,
            "separator": false,
            "spacing": "default",
            "type": "Container",
            "items": [
                {
                    "id": "",
                    "isVisible": false,
                    "separator": false,
                    "spacing": "default",
                    "type": "Container",
                    "items": [
                        {
                            "id": "",
                            "isVisible": true,
                            "separator": false,
                            "spacing": "default",
                            "type": "TextBlock",
                            "textStyle": "default",
                            "wrap": true,
                            "size": "small"
                        }
                    ],
                    "verticalContentAlignment": "center",
                    "bleed": false
                },
                {
                    "id": "",
                    "isVisible": true,
                    "separator": false,
                    "spacing": "default",
                    "type": "TextBlock",
                    "textStyle": "default",
                    "wrap": true,
                    "text": "This app clears your calendar but doesn’t report time off to **time and recording**. Select your dates."
                },
                {
                    "id": "startDate",
                    "isVisible": true,
                    "separator": false,
                    "spacing": "default",
                    "type": "Input.Date",
                    "isRequired": true,
                    "value": "2022-09-27",
                    "label": "Start date",
                    "errorMessage": "Enter a start date or select from calendar."
                },
                {
                    "id": "endDate",
                    "isVisible": true,
                    "separator": false,
                    "spacing": "default",
                    "type": "Input.Date",
                    "isRequired": true,
                    "value": "2022-09-27",
                    "label": "End date",
                    "errorMessage": "Enter an end date or select from calendar."
                },
                {
                    "id": "",
                    "isVisible": false,
                    "separator": false,
                    "spacing": "default",
                    "type": "TextBlock",
                    "textStyle": "default",
                    "wrap": false,
                    "text": "Enter an end date that is after start date.",
                    "color": "attention",
                    "size": "small",
                    "weight": "bolder"
                }
            ],
            "verticalContentAlignment": "center",
            "bleed": false
        },
        {
            "id": "",
            "isVisible": true,
            "separator": false,
            "type": "Container",
            "items": [],
            "verticalContentAlignment": "center",
            "bleed": false
        },
        {
            "id": "",
            "isVisible": true,
            "separator": false,
            "spacing": "default",
            "type": "ActionSet",
            "actions": [
                {
                    "style": "positive",
                    "id": "next-button",
                    "title": "Next",
                    "altText": "Next",
                    "type": "Action.Submit",
                    "associatedInputs": "auto",
                    "data": {
                        "id": "next-button"
                    }
                }
            ]
        }
    ],
    "actions": [],
    "refresh": null
}
```

**Comment by anna-dingler** [comment]
Hi @Raisul123,

I wasn't able to repro these issues on the given test site, so I am closing the bug. Please let me know if you have any questions for if the issue appears again.

Thanks!

**Comment by vagpt** [comment]
This issue is not repro at our end. Please refer to the below snippet for the reference.

**Snippet:**
![image](https://user-images.githubusercontent.com/95913748/235132619-72879555-0a61-4895-bddb-923b99aee43e.png)

![image](https://user-images.githubusercontent.com/95913748/235132675-b8e1eff9-2086-4e3c-897e-99ddeedad25b.png)
